core/src/main/scala-2/caliban/parsing/Parser.scala

```diff
@@ -1,9 +1,13 @@
 package caliban.parsing
 
 import caliban.CalibanError.ParsingError
+import caliban.InputValue
 import caliban.parsing.adt._
 import fastparse._
-import zio.{ IO, Task }
+import zio.IO
```

Comment:
I liked them grouped better :P

Reply:
Sorry, that was metals `Optimize imports`. I'll merge them again :)